ssg/yaml.py

```diff
@@ -46,6 +47,12 @@ def _get_implied_properties(existing_properties):
             "pkg_system" not in existing_properties):
         pkg_manager = existing_properties["pkg_manager"]
         result["pkg_system"] = PKG_MANAGER_TO_SYSTEM[pkg_manager]
+
+    if "uid_min" not in existing_properties:
+        result["uid_min"] = result["auid"] = DEFAULT_UID_MIN
+    else:
+        result["auid"] = existing_properties["uid_min"]
```

Comment:
This is doing two things, setting uid_min if it's not set in product and then setting auid according to uid_min. It doesn't respect any auid override so it's not clear to me why there even are 2 variables. Regardless, the result["auid"] logic should be a separate line. Handle uid_min, then handle auid, ...

Reply:
I have updated the logic to not override `auid` if it is already set.